packages/merge-styles/etc/merge-styles.api.md

```diff
@@ -411,13 +411,34 @@ export interface IStyleSheetConfig {
     injectionMode?: InjectionMode;
     namespace?: string;
     onInsertRule?: (rule: string) => void;
+    rtl?: boolean;
 }
 
 // @public
 export function keyframes(timeline: {
     [key: string]: {};
 }): string;
 
+// Warning: (ae-forgotten-export) The symbol "IStyleOptions" needs to be exported by the entry point index.d.ts
+// 
+// @public
+export function mergeCss(args: (IStyle | IStyleBaseArray | false | null | undefined) | (IStyle | IStyleBaseArray | false | null | undefined)[], options?: IStyleOptions): string;

```

Comment:
Should this just be "css" and the other "cssSet"? This would bring it more inline with other libraries:

```tsx
css({ background: 'red' });
cssSet({
  root: { background: 'red' },
  foo: { background: 'green' }
})